pyproject.toml

```diff
@@ -62,7 +65,14 @@ python_files = ["test_*.py", "*_test.py", "tests.py"]
 [tool.coverage.run]
 branch = true
 source = ["src"]
-omit = ["*/.tox/*", "*/__main__.py", "*/setup.py", "*/.venv*/*", "*/venv*/*"]
+omit = [
+  "*/.tox/*",
+  "*/__main__.py",
+  "*/setup.py",
+  "*/.venv*/*",
+  "*/venv*/*",
+  "src/nina/version.py"
```

Comment:
@tomrtk Okay using a direct path instead of any wildcard stuff?

Reply:
Think so.